adoc/deployment-sysreqs.adoc

```diff
@@ -27,6 +27,9 @@ The minimal viable failure tolerant production environment configuration consist
 * 3 master nodes
 * 2 worker nodes
 
+When cluster size not reaching desired number, the replica pods such as `oidc-dex` and `oidc-gangway`
```

Comment:
What does "desired number" mean here? Do I understand this correctly that the minimum configuration is not large enough to hold these replica pods?